### ISSUE TITLE
PhoneGap Desktop (0.2.1) - IOS Device Connection Issues

### ISSUE THREAD

**Issue by inex0rable** [phonegap/phonegap-app-desktop#523]
Hi all. Hope this helps.... of course guidance on a fix would be useful too.

Stuff:

    Windows 10 (pro 64bit - latest build, updates etc)
    PhoneGap Desktop (Downloaded today... version 0.2.1)

Devices:

    Samsung S4 mini - PhoneGap Developer App downloaded today from GooglePlay
    Apple iPhone 4s (ios 6.1.3) PhoneGap Developer App downloaded today from AppStore

Scenario:

    Installed desktop app on Windows 10 machine.
    Windows Firewall completely off.
    Created new PhoneGap sample test app thingy.
    Connected with desktop browsers - all works as expected. Cool.
    Connected with Samsung Device - all works as expected. Cool.
    Connected with iPhone - nothing happens (device lies that it is downloading).

Investigation:

    Fired up Wireshark / Fiddler to monitor traffic
    Notice the server log button in the phonegap desktop app - click it.
    Connected with iPhone - server log shows 404 /_ api _/appzip
    Use desktop browser to go to [IP Address]:3000/_ api _/appzip and get a 200 with zip file

Checked traffic:
    Desktop browser sends a GET request and gets a 200
    iPhone sends a HEAD request and gets a 404

Experimentation:
    Used Fiddler to replay the desktop browser request as a HEAD request - received the 404

Conclusion:
    HEAD requests are bad for PhoneGap Desktop but the ios app send them - and thus fails.

That's all I've got - have I missed something obvious? Is there a fix?
Thanks in advance in case anyone has a lead.

**Comment by hermwong** [comment]
I suspect that iOS 6.1.3 might only support an older version of the PhoneGap Developer App which isn't compatible with the latest version of the PhoneGap Desktop.

**Comment by ikivilev** [comment]
@hermwong Probably it was due to old iOS version (6.1.3). I tried Phonegap Developer App on iOS 9.2.1 and it worked well from the first try! However the first log record in Phonegap Desktop app was 404, and 200 after that.

![image](https://cloud.githubusercontent.com/assets/10121584/13026914/37445dea-d24b-11e5-96d1-ee4361f81046.png)

Is there a chance of running the link between iOS 6.1.3 and desktop?
